pathways/inspections.py

```diff
@@ -191,10 +337,16 @@ def shipment_infestation_rate(shipment):
     return count / shipment["num_stems"]
 
 
-def count_diseased(shipment):
+def count_diseased_boxes(shipment):
     """Return number of boxes with pest"""
     count = 0
     for box in shipment["boxes"]:
         if box:
             count += 1
     return count
+
+
+def count_diseased_stems(shipment):
+    """Return number of stems with pest"""
+    count = np.count_nonzero(shipment["stems"])
+    return count
```

Comment:
File should end with a new line character both in Git and in Python following PEP8. See if your editor supports something like PEP8, pep8, pycodestyle, pyflakes, ... to catch these automatically. The code in this repo is checked [using Flake8, Black and Pylint](https://github.com/ncsu-landscape-dynamics/pathways-simulation/blob/master/.github/workflows/ci.yml) with configuration [in the root directory](https://github.com/ncsu-landscape-dynamics/pathways-simulation). However, don't worry about it too much right now. I think I will just merge this PR once we are happy with the code and we can work on fixing it later (Black will fix the whitespace formalities automatically).

Reply:
Yes, the IDE I'm using has Pylint. Just went through with the linter and caught a bunch of bugs... Very helpful! There are still some issues (hopefully minor) I was not sure how to fix.